opentechinstitute/commotion-service-manager
# Dont remove local

includes https://github.com/opentechinstitute/luci-commotion-apps/pull/55

to test: 

0) flash two nodes
1) create application, make sure it shows up on neighbor node
2) edit the app, and make sure the changes show up immediately on the neighbor node
